### PR TITLE
Feature/hono influxdb connector update

### PR DESCRIPTION
**Pull request by unly**
update of the hono to influxdb connector to use the latest version of Hono and make it more stable.

### CODE REVIEW

File: utils/hono-influxdb-connector/src/main/java/org/eclipse/kuksa/honoConnector/HonoConnector.java [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+package org.eclipse.kuksa.honoConnector;
+
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Future;
+import io.vertx.core.Handler;
+import io.vertx.core.Vertx;
+import io.vertx.core.json.DecodeException;
+import io.vertx.core.json.JsonObject;
+import io.vertx.proton.ProtonClientOptions;
+import org.apache.qpid.proton.message.Message;
```
Comment (rohoet) [inline utils/hono-influxdb-connector/src/main/java/org/eclipse/kuksa/honoConnector/HonoConnector.java]: Third party content.

**Comment by unly** [comment]
@rohoet same as in PR #50 the dependencies are tracked in:

- https://github.com/eclipse/kuksa.cloud/pull/51/files#diff-25ea7d03d5b1b136ae6bfc980d923f9e

**Comment by rohoet** [comment]
@unly I see the gradle dependencies which look fine, however `HonoConnector.java` imports `org.apache.qpid.proton.message.Message;` which looks third party to me - or do I miss something?

**Comment by unly** [comment]
hey @rohoet  this should be the list of dependencies used for this connector. The `Message` class you mentioned is part of `org.apache.qpid:proton-j:0.25.0`.

- ch.qos.logback:logback-classic:1.2.3
- ch.qos.logback:logback-core:1.2.3
- com.fasterxml.jackson.core:jackson-annotations:2.9.0
- com.fasterxml.jackson.core:jackson-core:2.9.6
- com.fasterxml.jackson.core:jackson-databind:2.9.6
- com.jayway.jsonpath:json-path:2.4.0
- com.squareup.moshi:moshi:1.5.0
- com.squareup.okhttp3:logging-interceptor:3.11.0
- com.squareup.okio:okio:1.14.0
- com.squareup.retrofit2:converter-moshi:2.4.0
- com.squareup.retrofit2:retrofit:2.4.0
- com.vaadin.external.google:android-json:0.0.20131108.vaadin1
- io.jsonwebtoken:jjwt:0.7.0
- io.netty:netty-buffer:4.1.19.Final
- io.netty:netty-codec:4.1.19.Final
- io.netty:netty-codec-dns:4.1.19.Final
- io.netty:netty-codec-http2:4.1.19.Final
- io.netty:netty-codec-http:4.1.19.Final
- io.netty:netty-codec-socks:4.1.19.Final
- io.netty:netty-common:4.1.19.Final
- io.netty:netty-handler:4.1.19.Final
- io.netty:netty-handler-proxy:4.1.19.Final
- io.netty:netty-resolver:4.1.19.Final
- io.netty:netty-resolver-dns:4.1.19.Final
- io.netty:netty-transport:4.1.19.Final
- io.opentracing:opentracing-api:0.31.0
- io.opentracing:opentracing-noop:0.31.0
- io.vertx:vertx-core:3.5.4
- io.vertx:vertx-proton:3.5.4
- javax.annotation:javax.annotation-api:1.3.2
- javax.xml.bind:jaxb-api:2.2.12
- junit:junit:4.12
- net.bytebuddy:byte-buddy:1.9.3
- net.bytebuddy:byte-buddy-agent:1.9.3
- net.minidev:accessors-smart:1.2
- net.minidev:json-smart:2.3
- org.apache.logging.log4j:log4j-api:2.11.2
- org.apache.logging.log4j:log4j-to-slf4j:2.11.2
- org.apache.qpid:proton-j:0.25.0
- org.assertj:assertj-core:3.11.1
- org.eclipse.hono:hono-client:0.8
- org.eclipse.hono:hono-core:0.8
- org.hamcrest:hamcrest-core:1.3
- org.hamcrest:hamcrest-library:1.3
- org.influxdb:influxdb-java:2.14
- org.mockito:mockito-core:2.23.4
- org.msgpack:msgpack-core:0.8.16
- org.objenesis:objenesis:2.6
- org.ow2.asm:asm:5.0.4
- org.skyscreamer:jsonassert:1.5.0
- org.slf4j:jul-to-slf4j:1.7.25
- org.slf4j:slf4j-api:1.7.25
- org.springframework.boot:spring-boot:2.1.3.RELEASE
- org.springframework.boot:spring-boot-autoconfigure:2.1.3.RELEASE
- org.springframework.boot:spring-boot-starter:2.1.3.RELEASE
- org.springframework.boot:spring-boot-starter-logging:2.1.3.RELEASE
- org.springframework.boot:spring-boot-starter-test:2.1.3.RELEASE
- org.springframework.boot:spring-boot-test:2.1.3.RELEASE
- org.springframework.boot:spring-boot-test-autoconfigure:2.1.3.RELEASE
- org.springframework.security:spring-security-crypto:4.2.8.RELEASE
- org.springframework:spring-aop:5.1.5.RELEASE
- org.springframework:spring-beans:5.1.5.RELEASE
- org.springframework:spring-context:5.1.5.RELEASE
- org.springframework:spring-core:5.1.5.RELEASE
- org.springframework:spring-expression:5.1.5.RELEASE
- org.springframework:spring-jcl:5.1.5.RELEASE
- org.springframework:spring-test:5.1.5.RELEASE
- org.xmlunit:xmlunit-core:2.6.2
- org.yaml:snakeyaml:1.23

**Pull request merged**